native/src/routes/SearchModal.tsx

```diff
@@ -49,8 +50,10 @@ const SearchModal = ({
   const resourceCache = useResourceCache({ cityCode, languageCode })
   const { t } = useTranslation('search')
 
-  const contentLanguageResults = useSearch(documents, query)
-  const fallbackLanguageResults = useSearch(fallbackLanguageDocuments, query)
+  const debouncedQuery = useDebounce(query, DEBOUNCED_QUERY_TIMEOUT)
```

Comment:
:wrench: I think it would be nice to use `DEBOUNCED_QUERY_TIMEOUT` as default, such that there is no need to import/explicitly pass this here.
```suggestion
  const debouncedQuery = useDebounce(query)
```